Lib/traceback.py

```diff
@@ -84,8 +84,11 @@ def extract_tb(tb, limit=None):
     "another exception occurred:\n\n")
 
 
-_sentinel = object()
+class _Sentinel:
+    def __repr__(self):
+        return "<optional>"
```

Comment:
Maybe "<derived>" (or something else?) instead of "<optional>" .

Reply:
How about "<not specified>" or "<unspecified>"?